qiskit/opflow/evolutions/__init__.py

```diff
@@ -35,43 +35,43 @@
     equivalent to e^iHt.
 
 Evolution Base Class
-====================
+++++++++++++++++++++
```

Comment:
Is there a reason to use `+` instead of the normal subsection `-`? (https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections)

Reply:
Hmm, I thought I was being consistent with the rest of Terra, but by a real fluke, the only places I compared against were `circuit.library`, `algorithms` and `opflow`, which are the _only_ places that use `+`!  I'll swap it over - better to be consistent everywhere.